requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
-better_exceptions
```

Comment:
I used it for both, but now that we have bugsnag it's not really necessary in production. We can add it to the list of "useful things to have from a dev standpoint". It cleans up tracebacks and tells you what's in every variable at the time of the explosion.